Emby.Naming/Video/StackResolver.cs

```diff
@@ -21,31 +21,20 @@ public StackResolver(NamingOptions options)
 
         public IEnumerable<FileStack> ResolveDirectories(IEnumerable<string> files)
         {
-            return Resolve(files.Select(i => new FileSystemMetadata
-            {
-                FullName = i,
-                IsDirectory = true
-            }));
+            return Resolve(files.Select(i => new FileSystemMetadata { FullName = i, IsDirectory = true }));
         }
 
         public IEnumerable<FileStack> ResolveFiles(IEnumerable<string> files)
         {
-            return Resolve(files.Select(i => new FileSystemMetadata
-            {
-                FullName = i,
-                IsDirectory = false
-            }));
+            return Resolve(files.Select(i => new FileSystemMetadata { FullName = i, IsDirectory = false }));
         }
 
         public IEnumerable<FileStack> ResolveAudioBooks(IEnumerable<FileSystemMetadata> files)
         {
-            foreach (var directory in files.GroupBy(file => file.IsDirectory ? file.FullName : Path.GetDirectoryName(file.FullName)))
+            foreach (var directory in files.GroupBy(file =>
+                file.IsDirectory ? file.FullName : Path.GetDirectoryName(file.FullName)))
```

Comment:
IMO it would be more readable to pull this out into a variable above instead of having it all inside the `for` statement